### ISSUE TITLE
HttpPlayback dependency

### ISSUE THREAD

**Issue by alushin** [Garethp/php-ews#61]
The dependency on HttpPlayback forces you to set minimum stability to dev, while this is not the end of the world I would like to ask you to create a release for the HttpPlayback package.

**Comment by Garethp** [comment]
Good point. I think HttpPlayback is stable enough to consider releasing it. Expect this some time in the next week
